core/src/main/java/org/mule/runtime/core/internal/message/DefaultMessageBuilder.java

```diff
@@ -554,11 +559,15 @@ protected void serializeValue(ObjectOutputStream out) throws Exception {
       } else {
         out.writeBoolean(false);
         // TODO MULE-10013 remove this logic from here
```

Comment:
update references to this issue

Reply:
This reference is still correct isn't it?